bedrock/mozorg/helpers/misc.py

```diff
@@ -462,8 +462,10 @@ def absolute_url(url):
 
     if url.startswith('//'):
         prefix = 'https:'
-    else:
+    elif url.startswith('/'):
```

Comment:
Umm, the added `test_full` didn't make sense. I have added `test_urls` instead to check URLs as intended.